ui-v2/app/templates/dc/nodes/index.hbs

```diff
@@ -35,7 +35,7 @@
             <h2>Healthy Nodes</h2>
             {{#list-collection
                 items=healthy
-                cell-layout=(percentage-columns-layout healthy.length columns 100) as |item index|
+                cell-layout=(percentage-columns-layout healthy.length columns 92) as |item index|
```

Comment:
Why did this change?

Reply:
I noticed the vertical margins between the unhealthy nodes and the healthy nodes where different by 4px (22px vs 26px), so I equalled them up.

Before:

![before-2](https://user-images.githubusercontent.com/554604/42681109-bdc849a0-867e-11e8-88a2-784ac52bd5d9.png)

After:

![after-2](https://user-images.githubusercontent.com/554604/42681118-c28c536e-867e-11e8-8e35-002de4046de8.png)

Ideally they'd be the same, which reminds me I need to ask @opihana about the vertical rhythm of everything across the entire app